code/modules/mob/mob_helpers.dm

```diff
@@ -251,7 +251,7 @@
 	return copytext(sanitize(t),1,MAX_MESSAGE_LEN)
 
 /proc/derpspeech(message, stuttering)
-	message = replacetext(message, " am ", " ")

```

Comment:
You didn't actually remove the memes, this way the people will talk normally and sometimes yell a meme, instead of the current talking strange and yelling memes. 
Check out this line. On further though, the brain healing thing could go here instead.
https://github.com/tgstation/-tg-station/blob/97f9d909f1aad6a9cf1e99389d64f29ad5e01072/code/modules/mob/living/carbon/human/life.dm#L77